pkg/apis/machineconfiguration.openshift.io/v1/types.go

```diff
@@ -61,6 +61,9 @@ type MCOConfigSpec struct {
 	BaseDomain string `json:"baseDomain"`
 
 	SSHKey string `json:"sshKey"`
+
+	// Sourced from configmap/machine-config-osimageurl
+	OSImageURL string `json:"osImageURL`
```

Comment:
Doesn't seem like we need this, right?

Reply:
@jlebon as in the MC no longer needs to house an `OSImageURL`?

Reply:
> Doesn't seem like we need this, right?

Yep, you're right; the SSH key comes from the installconfig, but the `OSImageURL` doesn't.

> @jlebon as in the MC no longer needs to house an OSImageURL?

This is the `mcoconfig` object.